rasa_core/policies/keras_policy.py

```diff
@@ -55,10 +67,21 @@ def __init__(self,
 
         self.current_epoch = current_epoch
 
+    @staticmethod
+    def _load_tf_config(config: Dict[Text, Any]) -> None:
+        tf_config = tf.ConfigProto(
+            device_count={'CPU': config['device_count']},
+            inter_op_parallelism_threads=config['inter_op_threads'],
+            intra_op_parallelism_threads=config['intra_op_threads'],
+            gpu_options={'allow_growth': config['allow_growth']}
+        )
+        return tf_config
+
     def _load_params(self, **kwargs: Dict[Text, Any]) -> None:
         config = copy.deepcopy(self.defaults)
```

Comment:
self.config = copy.deepcopy(self.defaults).update(kwargs)

Reply:
As you mean, I should duplicate define `config` parameter, just like
`def _load_params(self, **kwargs: Dict[Text, Any]) -> None:
        config = copy.deepcopy(self.defaults)
        config.update(kwargs)

        self.config = copy.deepcopy(self.defaults).update(kwargs)
        self.rnn_size = config['rnn_size']
        self.epochs = config['epochs']
        self.batch_size = config['batch_size']
        self.validation_split = config['validation_split']`

Reply:
and `copy.deepcopy(self.defaults).update(kwargs)` return is None. `update` function hasn't return value

Reply:
sorry my bad, I meant if we need `config` to be class variable than there is no point to create local `config`. Because before you persisted the whole config instead of tf_config.

`tf.ConfigProto` can be persisted with pickle?

Reply:
yes, `tf.ConfigProto` can be presisted with pickle. And I did in `embedding_policy`, but I think it's no need to do that in `keras_policy`. It saves the parameter `config` json, and save in `keras_policy.json`

Reply:
then let's save the config in embedding policy as well